packages/requisitions/src/RequestRequisition/ListView/ListView.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { FC, useCallback } from 'react';
+import React, { FC, useCallback, useEffect } from 'react';
```

Comment:
requisitions don't have a colour (I reckon they should.. why not?!) so the row is disabled for deleting, but the gray seems even more wrong here